feat(spell-description): Add hover over spells

felipebrunetti94 opened the pull request:
![Gravação de tela de 14-11-2019 11_41_14](https://user-images.githubusercontent.com/47053687/68867092-0c744000-06d4-11ea-9964-114ac9c99de9.gif)
- Add hover over the menu area
- Still need the spells descriptions

Fix #15

felipebrunetti94 committed:
feat(spell-description): Add hover over spells

- Add hover over the menu area
- Still need the spells descriptions

Fix #15

negreirosleo commented on index.html:
```diff
@@ -61,5 +61,29 @@
\ No newline at end of file
+  initSounds();
+  
+var gameCanvas = document.getElementById("gameCanvas");
```
var? :thinking:

negreirosleo replied:
Btw, dont use ID's for selectors, you should use data-attributes instead.

felipebrunetti94 replied:
data attributes ? How exactly, could you give me an example ?

negreirosleo commented on js/game.js:
```diff
@@ -56,13 +58,26 @@ function draw() {
+
+function drawSpells() {
+  
+  for(let i=0; i<player.spells.length; i++){
```
How about a `forEach` here since player.spells is an array?

felipebrunetti94 replied:
Sure, I forgot about `foreach` kkk

negreirosleo commented on js/game.js:
```diff
@@ -56,13 +58,26 @@ function draw() {
 }
 
+
+function drawSpells() {
```
Use arguments 

```js
function drawSpells(player,hoverMenu) {
```

felipebrunetti94 replied:
good one

negreirosleo commented on js/game.js:
```diff
@@ -56,13 +58,26 @@ function draw() {
+    
+    if(hoverMenu)
+    {
+      drawText(`Description`, 20, false, 120+i*40, "aqua");
```
What does `120+i*40` mean? can this calculation be assigned to a variable?

```js
const calculationMeaning = 120+i*40
```

negreirosleo commented on index.html:
```diff
@@ -61,5 +61,29 @@
+  }        
+}
+
+gameCanvas.addEventListener('mousemove', () => mouseTracker(), true)
```
Why is this in the html? how about creating a file for this?

nuzcraft commented:
This is awesome!! Do you still have more changes to make, or is this good to merge?

felipebrunetti94 commented:
I was thinking in a visual update, like: "out of spells" color in read or blinking, strong spells in bold, things like that...if you are okay with it I can make the changes, or we can merge and open another issue, for a visual update.

felipebrunetti94 committed:
fix(spell-description): Fix changes in last commit

- change var for const
- change for to forEach
- change getElementById for querySelector

felipebrunetti94 commented:
@nuzcraft  Now it is ready for merge :D
